services/kafka/cluster.go

```diff
@@ -31,7 +30,7 @@ func overrideClientFromFileConfig(client *Client, productName, location string)
 		log.Printf("[WARN] Missing endpoint for %s in location %s", productName, location)
 		return
 	}
-	config.Servers = kafka.ServerConfigurations{
```

Comment:
@rmocanu-ionos is the `todo` for this function properly done? Check the comment above the function signature.

Reply:
done